src/Security/src/Authentication.CloudFoundryCore/CloudFoundryOpenIdConnectOptions.cs

```diff
@@ -41,14 +41,20 @@ public CloudFoundryOpenIdConnectOptions()
         /// </summary>
         public bool ValidateCertificates { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the timeout (in ms) for calls to the auth server
+        /// </summary>
+        public int Timeout { get; set; } = 100000;
```

Comment:
This change (combined with line 57) is likely to result in a higher timeout for at least one interaction